docs/examples/arrow_example.ipynb

```diff
@@ -92,6 +107,166 @@
+    "        else:\n",
+    "            schemas.append((attribute, pa.from_numpy_dtype(dtype)))\n",
+    "    return pa.schema(schemas)\n",
+    "\n",
+    "\n",
+    "print(\"-------node scehma-------\")\n",
+    "print(pgm_schema(DatasetType.input, ComponentType.node))\n",
+    "print(\"-------asym load scehma-------\")\n",
+    "print(pgm_schema(DatasetType.input, ComponentType.asym_load))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Schema with asymmetric attributes as a struct\n",
+    "\n",
+    "The phases can also be combined together in a pyarrow fixed size list."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-------node combined asym scehma-------\n",
+      "id: int32\n",
+      "u_rated: double\n",
+      "-------asym load combined asym scehma-------\n",
+      "id: int32\n",
+      "node: int32\n",
+      "status: int8\n",
+      "type: int8\n",
+      "p_specified: fixed_size_list<item: double>[3]\n",
+      "  child 0, item: double\n",
+      "q_specified: fixed_size_list<item: double>[3]\n",
+      "  child 0, item: double\n"
+     ]
+    }
+   ],
+   "source": [
+    "def pgm_combined_schema(dataset_type: DatasetType, component_type: ComponentType):\n",
+    "    schemas = []\n",
+    "    component_dtype = power_grid_meta_data[dataset_type][component_type].dtype\n",
+    "    for attribute, (dtype, _) in component_dtype.fields.items():\n",
+    "        if dtype.shape == (3,):\n",
+    "            pa_dtype = pa.list_(pa.from_numpy_dtype(dtype.base), 3)\n",
+    "        else:\n",
+    "            pa_dtype = pa.from_numpy_dtype(dtype)\n",
+    "        schemas.append((attribute, pa_dtype))\n",
+    "    return pa.schema(schemas)\n",
+    "\n",
+    "\n",
+    "print(\"-------node combined asym scehma-------\")\n",
+    "print(pgm_combined_schema(DatasetType.input, ComponentType.node))\n",
+    "print(\"-------asym load combined asym scehma-------\")\n",
+    "print(pgm_combined_schema(DatasetType.input, ComponentType.asym_load))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "pyarrow.RecordBatch\n",
+       "id: int32\n",
+       "u_rated: double\n",
+       "----\n",
+       "id: [1,2,3]\n",
+       "u_rated: [10500,10500,10500]"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#TODO Decisions: Create from schema\n",
+    "pa.record_batch(\n",
+    "    {\n",
+    "        \"id\": [1, 2, 3],\n",
+    "        \"u_rated\": [10500.0, 10500.0, 10500.0],\n",
+    "    },\n",
+    "    schema=pgm_schema(DatasetType.input, ComponentType.node),\n",
```

Comment:
> Table creation should be individually or via the schema?

I don't fully understand what you mean but I suppose you mean whether to use `power_grid_meta_data`? if so: yes, please do use that to deduce the data types

Reply:
Yes, table creation by power_grid_meta_data instead of manually specifying types.
Ill change to use from meta.

Reply:
I changed the wording. I am not sure how to mention "guaranteed" when an advanced user can indeed guarantee it if custom types are built correctly. :D